src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -455,14 +455,16 @@ class SchedulerActions(
 
     }
 
-    instancesToStart.foreach { toStart: Int =>
```

Comment:
why not use foreach? :) I think it's the way we do such stuff in scala, right?

Reply:
Async/await cannot be nested in closures.

Reply:
Exactly!